registration/forms.py

```diff
@@ -355,6 +385,41 @@ def update_user(self, application, commit=True):
                 application.user.profile.save()
                 application.save()
 
+        if self.restart_fields_changed():
+            messages.add_message(self.request, messages.INFO,
+                                 "Thank you for submitting these changes - we will rebuild your instance to "
+                                 "apply them, and email you to confirm once it is up to date.")
+
+            if settings.VARIABLES_NOTIFICATION_EMAIL:
+                subject = '{prefix} Update required at instance {name}'.format(
+                    prefix=settings.EMAIL_SUBJECT_PREFIX,
+                    name=application.subdomain,
+                )
+                text = ("Instance {instance_id} at {domain} requires a redeployment "
+                        "because some values were changed through the "
+                        "registration form.\n"
+                        "\n"
+                        "The new values are:\n"
+                        "- main color: {main_color}\n"
+                        "- link color: {link_color}\n"
+                        "- primary background color: {bg_color_1}\n"
+                        "- secondary background color: {bg_color_2}\n"
+                        "- logo: {logo}\n"
+                        "- favicon: {favicon}\n"
+                       ).format(
+                           instance_id=application.instance_id,
+                           domain=application.domain,
+                           main_color=application.main_color,
+                           link_color=application.link_color,
+                           bg_color_1=application.bg_color_1,
+                           bg_color_2=application.bg_color_2,
+                           logo=application.logo.url,
+                           favicon=application.favicon.url,
+                       )
```

Comment:
@clemente Could you place this email template into a separate file, and then render it using the django template engine instead?